ui/package.json

```diff
@@ -6,11 +6,7 @@
     "lint": "eslint --ext .ts,.tsx,.js packages/*",
     "fix": "eslint --fix --ext .ts,.tsx,.js packages/*",
     "test": "jest --coverage --config jest.config.js",
-    "dev": "next dev",
-    "build": "next build",
```

Comment:
Do we really want to remove these entirely? :thinking:

Reply:
Yeah, these are just on the root `package.json`. Don't really serve any purpose there.